package.json

```diff
@@ -112,6 +112,10 @@
         "webpack-merge": "^5.8.0"
     },
     "dependencies": {
+        "@ethersproject/bignumber": "^5.7.0",
```

Comment:
could you delete the packages that are not in use?

Reply:
Used.